cmd/entrypoint/runner.go

```diff
@@ -161,19 +143,11 @@ func (rr *realRunner) Run(ctx context.Context, args ...string) error {
 	return nil
 }
 
-// newTeeReader creates a new Reader that copies data from the given pipe function
-// (i.e. cmd.StdoutPipe, cmd.StderrPipe) into a file specified by path.
-// The file is opened with os.O_WRONLY|os.O_CREATE|os.O_APPEND, and will not
+// newStdLogWriter create a new file writer that used for collecting std log
+// the file is opened with os.O_WRONLY|os.O_CREATE|os.O_APPEND, and will not
 // override any existing content in the path. This means that the same file can
-// be used for multiple streams if desired.
-// The behavior of the Reader is the same as io.TeeReader - reads from the pipe
-// will be written to the file.
-func newTeeReader(pipe func() (io.ReadCloser, error), path string) (*namedReader, error) {
-	in, err := pipe()
-	if err != nil {
-		return nil, fmt.Errorf("error creating pipe: %w", err)
-	}
-
+// be used for multiple streams if desired. note that close after use
```

Comment:
Is this docstring missing after "note that close after use"?

Reply:
It's not missing, I just modified the note here and added it to where it was used.
notes related to teaReader are no longer needed here